.github/workflows/TICKET-MATE-BE-CD.yaml

```
@@ -31,7 +31,12 @@ jobs:
       # firebase 서비스워커 파일을 빌드 전에 생성
       - name: Create firebase-messaging-sw.js from secret
         run: |
-          echo "${{ secrets.FIREBASE_MESSAGING_SW_JS }}" > ./src/main/resources/static/firebase-messaging-sw.js          
+          echo "${{ secrets.FIREBASE_MESSAGING_SW_JS }}" > ./src/main/resources/static/firebase-messaging-sw.js     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)